app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -21,28 +21,18 @@ class DisabilityCompensationJob
 
     def perform(form526_submission_id, full_name)
       form526_submission = Form526Submission.find(form526_submission_id)
-      icn = Account.where(idme_uuid: form526_submission.user_uuid).first.icn
-
-      client = Lighthouse::VeteransHealth::Client.new(icn)
-      observations_response = client.get_resource('observations')
-      medicationrequest_response = client.get_resource('medications')
+      client = Lighthouse::VeteransHealth::Client.new(get_icn(form526_submission))
```

Comment:
This isn't related to your updates, but I was wondering to myself yesterday if there's any reason we don't move both of these lines into the rescue block also?

Reply:
I kept them out because if the api is having intermittent outages we want the job to retry but we don't care as much about those errors and they might be burdensome.